### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by hantsy** [NativeScript/nativescript-cli#4032]
**Environment**
C:\WINDOWS\system32>npm -v
6.0.0

C:\WINDOWS\system32>node -v
v10.12.0

**Describe the bug**
npm i -g nativescript
npm ERR! code ERR_STREAM_WRITE_AFTER_END
npm ERR! write after end
npm ERR! code ERR_STREAM_WRITE_AFTER_END
npm ERR! write after end

npm ERR! A complete log of this run can be found in:
npm ERR!     E:\Users\hantsy\AppData\Roaming\npm-cache\_logs\2018-10-15T13_33_24_148Z-debug.log

In this **debug.log** file, I found something like the following info:

```
8956 silly tarball no local data for supports-color@5.4.0. Extracting by manifest.
8957 verbose stack Error [ERR_STREAM_WRITE_AFTER_END]: write after end
8957 verbose stack     at writeAfterEnd (_stream_writable.js:243:12)
8957 verbose stack     at PassThrough.Writable.write (_stream_writable.js:291:5)
8957 verbose stack     at PassThrough.Writable.end (_stream_writable.js:579:10)
8957 verbose stack     at ReadEntry.entry.on (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\pacote\lib\extract-stream.js:19:41)
8957 verbose stack     at ReadEntry.emit (events.js:187:15)
8957 verbose stack     at ReadEntry.emit (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\node_modules\minipass\index.js:296:25)
8957 verbose stack     at ReadEntry.[maybeEmitEnd] (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\node_modules\minipass\index.js:249:12)
8957 verbose stack     at ReadEntry.end (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\node_modules\minipass\index.js:162:27)
8957 verbose stack     at Unpack.[consumeBody] (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\lib\parse.js:210:13)
8957 verbose stack     at Unpack.[consumeChunkSub] (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\lib\parse.js:391:40)
8957 verbose stack     at Unpack.[consumeChunk] (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\lib\parse.js:362:30)
8957 verbose stack     at Unzip.(anonymous function).on.chunk (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\lib\parse.js:291:59)
8957 verbose stack     at Unzip.emit (events.js:182:13)
8957 verbose stack     at Unzip.emit (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\node_modules\minipass\index.js:296:25)
8957 verbose stack     at Unzip.write (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\node_modules\minipass\index.js:99:17)
8957 verbose stack     at Unzip.write (E:\Users\hantsy\AppData\Roaming\npm\node_modules\npm\node_modules\tar\node_modules\minizlib\index.js:284:29)
8958 verbose cwd C:\WINDOWS\system32
8959 verbose Windows_NT 10.0.17134
8960 verbose argv "C:\\Program Files\\nodejs\\node.exe" "E:\\Users\\hantsy\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "i" "-g" "nativescript"
8961 verbose node v10.12.0
8962 verbose npm  v6.0.0
8963 error code ERR_STREAM_WRITE_AFTER_END
8964 error write after end
8965 verbose exit [ 1, true ]
```

I have tried to use `npm cache clean --force `, and reinstall it, it does not work.

**Comment by KristianDD** [comment]
Hi @hantsy ,
Similar issue is logged [here ](https://github.com/npm/npm/issues/19989) and according to the last comment it should be fixed in `npm@6.1.0`.

Please try updating to latest npm - `npm install npm -g` and check if the `nativescript` package installation completes successfully.

**Comment by hantsy** [comment]
@KristianDD Fixed it by upgrading NPM. Thanks
